napari/layers/labels/_tests/test_labels.py

```diff
@@ -754,6 +754,29 @@ def test_value():
     assert value == data[0, 0]
 
 
+@pytest.mark.parametrize(
+    'position,view_direction,dims_displayed,world',
+    [
+        ([10, 5, 5], [1, 0, 0], [0, 1, 2], False),
+        ([10, 5, 5], [1, 0, 0], [0, 1, 2], True),
+        ([0, 10, 5, 5], [0, 1, 0, 0], [1, 2, 3], True),
+    ],
+)
+def test_value_3d(position, view_direction, dims_displayed, world):
+    """Currently get_value should return None in 3D"""
```

Comment:
this doc seems wrong, copied from image layer?

```suggestion
    """get_value should return label value in 3D"""
```